examples/htool.c

```diff
@@ -1542,6 +1544,105 @@ static const struct htool_cmd CMDS[] = {
                  .desc="The signed cert file."},
                  {}},
     },
+    {
+        .verbs = (const char*[]){"security", "get_alias_key_v0_cert", NULL},
+        .desc = "Get the Alias Key V0 Cert",
+        .func = htool_get_alias_key_v0_cert,
+        .params =
+            (const struct htool_param[]){
+                {HTOOL_FLAG_VALUE, .name = "output", .default_value = "",
+                 .desc = "The Alias Key Certificate"},
+                {}},
+    },
+    {
+        .verbs = (const char*[]){"security", "get_alias_key_v1_cert", NULL},
+        .desc = "Get the Alias Key V1 Cert",
+        .func = htool_get_alias_key_v1_cert,
+        .params =
+            (const struct htool_param[]){
+                {HTOOL_FLAG_VALUE, .name = "output", .default_value = "",
+                 .desc = "The Alias Key Certificate"},
+                {}},
+    },
+
+    {
+        .verbs = (const char*[]){"security", "get_device_id_cert", NULL},
+        .desc = "Get the Device ID Cert",
+        .func = htool_get_device_id_cert,
+        .params =
+            (const struct htool_param[]){
+                {HTOOL_FLAG_VALUE, .name = "cert_output", .default_value = "",
+                 .desc = "The Device ID Certificate"},
+                {HTOOL_FLAG_VALUE, .name = "endorsement_cert_output",
+                 .default_value = "",
+                 .desc = "The Endorsement Device ID Certificate"},
+                {}},
+    },
+    {
+        .verbs = (const char*[]){"security", "get_attestation_pub_cert", NULL},
+        .desc = "Get the Attestation Public Cert",
+        .func = htool_get_attestation_pub_cert,
+        .params =
+            (const struct htool_param[]){
+                {HTOOL_FLAG_VALUE, .name = "output", .default_value = "",
+                 .desc = "The Public Certificate"},
+                {}},
+    },
+    {
+        .verbs = (const char*[]){"security", "get_signed_attestation_pub_cert",
+                                 NULL},
+        .desc = "Get the Signed Attestation Public Cert",
+        .func = htool_get_signed_attestation_pub_cert,
+        .params =
+            (const struct htool_param[]){
+                {HTOOL_FLAG_VALUE, .name = "output", .default_value = "",
+                 .desc = "The Signed Attestation Public Certificate"},
+                {}},
+    },
+    {
+        .verbs = (const char*[]){"tokens", "get_token_set_count", NULL},
```

Comment:
Rather than show three token commands, let's expose one command "htool security attestation" to get the whole attestation.